### DISCUSSION TITLE
Changing "Rooms" to "Houses"

### DISCUSSION THREAD

**Discussion by cannedcake** [benawad/dogehouse#2314] [Ideas]
Changing the name of rooms from Rooms to Houses.

Disclaimer:
This is just a random idea I thought of, and even though there are some problems with it I still want to see if anyone else likes this idea.

**Comment by redpwilliams** [comment]
I'm surprised it's been 4 days and no one else has commented. Seems like a brilliant idea that makes DogeHouse stand apart more from its competitors. Well done!

**Reply by cannedcake** [reply]
While it may make DogeHouse unique, since the brand name is DogeHouse it may refer to a large house filled of rooms (which are chatrooms).

This was one of the flaws with this idea.

**Reply by svalordev** [reply]
yeah that makes sense @cannedcake
